templates/themes/horizon/invenio_communities/details/home/index.html

```diff
@@ -48,19 +48,76 @@ <h1 class="ui large header">{{ _("Open repository for EU-funded research (Pilot)
         </div>
       </div>
     </div>
+
+  <!--BROWSE SECTION-->
+  {% macro format_metric_value(title, value) %}
+    {% if title == "total_records" or title == "total_grants" %}
+      {% if value >= 1000000 %}
+        {{ (value // 1000000) }} M
+      {% elif value >= 100000 %}
+        {{ '{:,.0f}'.format(value // 1000) }} K
+      {% else %}
+        {{ '{:,.0f}'.format(value) }}
+      {% endif %}
+    {% elif title == "total_data" %}
+      {% if value >= 1000 %}
+        {{ (value / 1000) | round(1) }} TB
+      {% else %}
+        {{ value }} GB
```

Comment:
```suggestion
        {{ (value // 1000000) }}M
      {% elif value >= 100000 %}
        {{ '{:,.0f}'.format(value // 1000) }}K
      {% else %}
        {{ '{:,.0f}'.format(value) }}
      {% endif %}
    {% elif title == "total_data" %}
      {% if value >= 1000 %}
        {{ (value / 1000) | round(1) }}TB
      {% else %}
        {{ value }}GB
```

To normalize with how we do it for views/downloads:

<img width="360" alt="image" src="https://github.com/zenodo/zenodo-rdm/assets/1814661/da06a50d-d52a-4fda-ab64-2d3dda0b494a">